feature_apps/src/main/java/cm/aptoide/pt/feature_apps/data/di/RepositoryModule.kt

```diff
@@ -78,4 +70,8 @@ internal object RepositoryModule {
       storeName = storeName
     )
   }
-}
\ No newline at end of file
+}
+
+@Qualifier
+@Retention(AnnotationRetention.BINARY)
+annotation class WidgetsService
```

Comment:
Why do we need this if we have a dedicated provides method?

Reply:
i thought that without this anotation it wouldnt be able to get it because i was getting a missing provides method error, but maybe it was due to another issue. Fixed 👍